### PR TITLE
chore(ReleaseInfo): update to v5.25.0

### PR DESCRIPTION
**Pull request by marstamm**
### Proposed Changes
Updates release info with latest changes:

image

related to https://github.com/camunda/camunda-modeler/issues/4363
 

### Checklist

To ensure you provided everything we need to look at your PR:

* [x] **Brief textual description** of the changes present
* [x] **Visual demo** attached
* [ ] **Steps to try out** present, i.e. [using the `@bpmn-io/sr` tool](https://github.com/bpmn-io/sr)
* [x] Related issue linked via `Closes {LINK_TO_ISSUE}` or `Related to {LINK_TO_ISSUE}`

### CODE REVIEW

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 1]
```diff
@@ -44,16 +44,12 @@ export function ReleaseInfo(props) {
     <div className={ css.ReleaseInfo }>
       <ul className="dashed">
         <li>
-          <h4>Properties panel quality of life improvements</h4>
-          We removed automatic (magic) sorting from all properties panels. Additional minor improvements ensure that implementing Camunda 7 and 8 is simpler than ever.
+          <h4>Enhanced Form Component Documentation</h4>
```
Comment (barmac) [inline client/src/plugins/version-info/ReleaseInfo.js:47]:
```suggestion
          <h4>Enhanced form components documentation</h4>
```

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 2]
```diff
@@ -44,16 +44,12 @@ export function ReleaseInfo(props) {
...
+          We've added convenient documentation links directly within the properties panel for Form components. This allows you to quickly find information about the functions and configurations when you need it.
         </li>
         <li>
-          <h4>More robust BPMN diagram import</h4>
-          Various improvements ensure that the modeler can import even what is (accidentally) broken.
-        </li>
-        <li>
-          <h4>New modeling experience inside</h4>
-          You can now enable our reworked context pad via <a href="https://docs.camunda.io/docs/next/components/modeler/desktop-modeler/flags/#enable-new-context-pad" target="_blank" rel="noopener">a feature toggle</a>.
+          <h4>Improved Accessibility</h4>
```
Comment (barmac) [inline client/src/plugins/version-info/ReleaseInfo.js:51]:
```suggestion
          <h4>Improved accessibility</h4>
```

Comment (barmac) [reply client/src/plugins/version-info/ReleaseInfo.js:51]: Sentence case

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 3]
```diff
@@ -44,16 +44,12 @@ export function ReleaseInfo(props) {
...
+          Our commitment to making the Modeler more accessible is ongoing. This release features enhanced keyboard navigation, screen-reader friendly labels, and better visual cues for links to assist users with impairments.
```
Comment (barmac) [inline client/src/plugins/version-info/ReleaseInfo.js:52]: I'd drop "with impairments". Accessibility improvements can serve anybody, and I'd like us to put emphasis on inclusiveness of the tool.

**Review by barmac — COMMENTED** [review]
I left a couple of comments to address but overall it looks good 👍

File: client/src/plugins/version-info/ReleaseInfo.js (hunk 3 above)
Comment (marstamm) [reply client/src/plugins/version-info/ReleaseInfo.js]: That makes sense. I changed `assist users with impairments` to `assist all users`

File: client/src/plugins/version-info/ReleaseInfo.js (hunk 1 above)
Comment (marstamm) [reply client/src/plugins/version-info/ReleaseInfo.js]: addressed in [049a158](https://github.com/camunda/camunda-modeler/pull/4384/commits/049a1588313c7b31edbecb51d039ecb4e7b4132a)

File: client/src/plugins/version-info/ReleaseInfo.js (hunk 2 above)
Comment (marstamm) [reply client/src/plugins/version-info/ReleaseInfo.js]: addressed in [049a158](https://github.com/camunda/camunda-modeler/pull/4384/commits/049a1588313c7b31edbecb51d039ecb4e7b4132a)

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 4]
```diff
@@ -44,16 +44,12 @@ export function ReleaseInfo(props) {
...
+          We've added convenient documentation links directly within the properties panel for Form components. This allows you to quickly find information about the functions and configurations when you need it.
         </li>
         <li>
-          <h4>More robust BPMN diagram import</h4>
-          Various improvements ensure that the modeler can import even what is (accidentally) broken.
-        </li>
-        <li>
-          <h4>New modeling experience inside</h4>
-          You can now enable our reworked context pad via <a href="https://docs.camunda.io/docs/next/components/modeler/desktop-modeler/flags/#enable-new-context-pad" target="_blank" rel="noopener">a feature toggle</a>.
+          <h4>Improved accessibility</h4>
```
Comment (nikku) [inline client/src/plugins/version-info/ReleaseInfo.js:51]:
```suggestion
          <h4>Accessibility improvements</h4>
```

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 5]
```diff
@@ -44,16 +44,12 @@ export function ReleaseInfo(props) {
     <div className={ css.ReleaseInfo }>
       <ul className="dashed">
         <li>
-          <h4>Properties panel quality of life improvements</h4>
-          We removed automatic (magic) sorting from all properties panels. Additional minor improvements ensure that implementing Camunda 7 and 8 is simpler than ever.
+          <h4>Enhanced form component documentation</h4>
+          We've added convenient documentation links directly within the properties panel for Form components. This allows you to quickly find information about the functions and configurations when you need it.
```
Comment (nikku) [inline client/src/plugins/version-info/ReleaseInfo.js:48]: What does this refer to?

Comment (marstamm) [reply client/src/plugins/version-info/ReleaseInfo.js:48]: Yes, this refers to the form-js feature you linked:
image

Comment (nikku) [reply client/src/plugins/version-info/ReleaseInfo.js:48]: So basically about the button to the documentation.

Comment (nikku) [reply client/src/plugins/version-info/ReleaseInfo.js:48]: We could consider to shorten it:

```suggestion
          Documentation links embedded into the properties panel give you quick access to functions and configurations each Form component supports.
```

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 6]
```diff
@@ -44,16 +44,12 @@ export function ReleaseInfo(props) {
...
         </li>
         <li>
-          <h4>More robust BPMN diagram import</h4>
-          Various improvements ensure that the modeler can import even what is (accidentally) broken.
-        </li>
-        <li>
-          <h4>New modeling experience inside</h4>
-          You can now enable our reworked context pad via <a href="https://docs.camunda.io/docs/next/components/modeler/desktop-modeler/flags/#enable-new-context-pad" target="_blank" rel="noopener">a feature toggle</a>.
+          <h4>Accessibility improvements</h4>
+          Our commitment to making the Modeler more accessible is ongoing. This release features enhanced keyboard navigation, screen-reader friendly labels, and better visual cues for links to assist all users.
```
Comment (nikku) [inline client/src/plugins/version-info/ReleaseInfo.js:52]: Consider to shorten this:

```suggestion
          This release enhances keyboard navigation, adds screen-reader friendly labels, and better visual cues for the benefit of our users.
```

File: client/src/plugins/version-info/ReleaseInfo.js [hunk 7]
```diff
@@ -44,16 +44,12 @@ export function ReleaseInfo(props) {
     <div className={ css.ReleaseInfo }>
       <ul className="dashed">
         <li>
-          <h4>Properties panel quality of life improvements</h4>
-          We removed automatic (magic) sorting from all properties panels. Additional minor improvements ensure that implementing Camunda 7 and 8 is simpler than ever.
+          <h4>Enhanced form component documentation</h4>
```
Comment (nikku) [inline client/src/plugins/version-info/ReleaseInfo.js:47]:
```suggestion
          <h4>Form component documentation</h4>
```

Comment (nikku) [reply client/src/plugins/version-info/ReleaseInfo.js:47]: (I'd make this a secondary item; I think accessibility is more important).

**Pull request merged**